stretch_demos/package.xml

```diff
@@ -81,6 +81,8 @@
   <exec_depend>tf</exec_depend>
   <exec_depend>tf2</exec_depend>
   <exec_depend>ros_numpy</exec_depend>
+  <build_depend>py_trees_ros</build_depend>
+  <build_depend>rqt_py_trees</build_depend>
 
```

Comment:
Add   `<exec_depend>message_generation</exec_depend>`  to build the new Action messages in runtime? While trying out the ArucoHeadScan action server previously, I required this addition.